nas.yml

```diff
@@ -130,3 +130,7 @@
   - import_tasks: tasks/get_iplayer.yml
     when: (get_iplayer_enabled | default(False))
     tags: get_iplayer
+
+  - import_tasks: tasks/jackett.yml
+    when: jackett_enabled
```

Comment:
Please can you make the `when:` clause include a default as with the other blocks

Reply:
@davestephens I have made this change, but I don't know how to update this pull request. Could you point me in the right direction?

Reply:
I have also updated the README with the following line, but for some reason it is not showing up here. 
 `* [Jackett] (https://github.com/Jackett/Jackett) - API Support for your favorite torrent trackers`

Git/Github is very confusing to me for some reason.

Reply:
Looks good to me - you can see all of the changes in the diff view: https://github.com/davestephens/ansible-nas/pull/67/files